src/configure.py.in

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
I would try to avoid putting any logic in this file, and instead just have it look something like
```py
DATADIR = """@DATADIR@"""
PACKAGE = """@PACKAGE@"""
```
Then import those in other files with `configure.DATADIR` etc.